layers/+lang/factor/funcs.el

```diff
@@ -8,11 +8,116 @@
 ;; This file is not part of GNU Emacs.
 ;;
 ;;; License: GPLv3
+(require 'loadhist)
+
+(defvar factor/ui-listener-root nil
+  "If set, commands that start the graphical listener will use
+this path to lok for the factor executable and fuel mode
+implementation. Intended to be customized per-project.")
+
+(defvar factor/ui-listener-args ""
+  "Extra arguments to the factor VM binary.")
 
 (defun factor//fuel-stack-effect ()
   "Small wrapper around factors stack effect help. If region is
-  active, use that, otherwise use sexp under point."
+active, use that, otherwise use sexp under point."
   (interactive)
   (if (region-active-p)
       (call-interactively 'fuel-stack-effect-region)
     (call-interactively 'fuel-stack-effect-sexp)))
+
+(defun factor//fuel-elisp-dir ()
```

Comment:
I found the instructions in `CONVENTIONS.org` to be confusing here:

```org
* Code guidelines
** Spacemacs core and layer
Function names follow these conventions:
- =spacemacs/xxx= is an interactive function called =xxx=
- =spacemacs//xxx= is a private function called =xxx= (implementation details)
- =spacemacs|xxx= is a /macro/ called =xxx=

Variables follow these conventions:
- =spacemacs-xxx= is a variable
- =spacemacs--xxx= is a private variable (implementation details)

** All layers
A package is initialized in a function with name =<layer>/init-xxx= where:
- =<layer>= is the layer name
- =xxx= is the package name
…
```

So does `Spacemacs core and layer` refer to every layer here (There is a section `All layers` after all, which does not contain specifics for function naming)?  I see some layers which prefix their functions in `funcs.el` as `layer/function` (e.g. `latex`), and some which prefix their functions as `spacemacs/layer-function` (e.g. `helm`), and some which seem to do both (e.g. `erc`).

Reply:
Hmmm, basically the convention for `Code Guidelines` for `spacemacs core and layer` apply to all layers. The ones you have found need to be adapted to follow the guideline. 

I'll make a note to check if there are more and fix them in time, thanks for pointing me there.

Reply:
@smile13241324 Since also the existing function names in this layer would have to be changed according to this convention, it might make sense to leave it like this for this PR and change all of them in a follow-up PR?

Reply:
Sounds good